http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2826,9 +2826,7 @@ components:
           type: integer
           description: The size of the persistent disk to be created in GB.
         diskType:
-          allOf:
```

Comment:
should this be `oneof`? 🤔

Reply:
No - it's an enum so it should just reference the enum type. If there were a more complex disk configuration per type, then you may want a oneof on StandardDiskConfig and SolidStateDiskConfig, using a discriminator of (still) the diskType enum.